packages/desktop-client/src/components/banksync/EditSyncAccount.tsx

```diff
@@ -200,10 +209,23 @@ export function EditSyncAccount({ account }: EditSyncAccountProps) {
       {({ state: { close } }) => (
         <>
           <ModalHeader
-            title={t('Account settings')}
+            title={account.name + t(' bank sync settings')}
```

Comment:
It makes it easier to translate if the variable is included in the translation string.

```suggestion
            title={t('{{accountName}} bank sync settings', { accountName: account.name })}
```